src/locale/kz.js

```diff
@@ -0,0 +1,30 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'kz',
```

Comment:
I know this is really strange. I don't know why its named as kk in momentjs. It doesn't feel right to me, because everything related to Kazakh language or Kazakhstan is associated with kz rather than kk. So I'd go with kz.

Reply:
After checking wikipedia, I found kz is two-letter country codes of Kazakhstan (ISO_3166-1), while kk is short code for language Kazakh (ISO_639-2).

Also I append this to #171 for future reference.

Reply:
Oh, shame on me I didn't know that. Should I update pr with kk?

Reply:
Yes please if you have time.